### ISSUE TITLE
Infinite problem with sftp is back

### ISSUE THREAD

**Issue by ArminDeeg** [miyako/4d-plugin-curl-v2#2]
![2019-06-11 13_43_18-EXPOSE 9](https://user-images.githubusercontent.com/6436037/59269348-1ee2dd00-8c4f-11e9-8776-ad09f2fd25bd.png)

Infinite problem with sftp is back

**Comment by ArminDeeg** [comment]
carbon branch 4d v16.x 32

**Comment by miyako** [comment]
can you please at least share basic facts, what do you mean "it's back" ?

did you update or replace the plugin or 4D or the system?

when was it OK and when did it change?

I have no idea what you are talking about


Keisuke Miyako
Technical Service Manager

4D Japan
1-10-2-6F Dogenzaka
150-0043 Shibuya Tokyo
Japan

Téléphone : 03-4400-1789
Standard :  03-4400-1789
Fax :       03-6427-8449
Email :     Keisuke.Miyako@4d.com
Web :       www.4D.com/jp

**Comment by ArminDeeg** [comment]
Sorry Miyako,

I've updated the curl v2 carbon plugin v2 from 3.1 to 3.2, cause all my macs with Mojave 10.4.5 stops working with curl problems.

4D v16.x 32 Bit Windows

I thought that you remember the previous bug posting from me, with curl_ftp and curl_v2 with the infinite file upload problems.

To resolve the problem, I've pasted the previous Windows and Windows64 folders from version 3.1 into the 3.2 curl v2 folder.

Armin
